src/components/results/candidate-results/personality-archetype/index.js

```diff
@@ -33,24 +34,56 @@ class PersonalityArchetype extends Component {
     const personality = this.props.assessment.archetype;
     if(!personality) { return null; }
     const badge = personality.details.find(({title}) => title === "Badge");
-    const description = personality.details.find(({title}) => title === "Candidate Description");
+    const perspective = this.props.getOption("perspective");
+    const description = (
+      perspective === "thirdPerson"
+        ? personality.details.find(({title}) => title === "Hiring Manager Description")
+        : personality.details.find(({title}) => title === "Candidate Description")
+    );
 
     return (
-      <div className={style.container}>
-        <div className={style.details}>
-          {badge && <img alt={personality.name} src={badge.body} />}
-          <DangerousHTML
-            html={this.props.translate("personality_heading", {
-              deck_name: this.props.deck.name,
-              personality: `<span>${personality.name}</span>`
-            })}
-            tag="h2"
-          />
-          {description && <p>{description.body}</p>}
-        </div>
-        <div className={style.meaning}>
-          <DangerousHTML html={this.props.translate("candidate_description_for_archetype_html")} tag="p" />
-        </div>
+      <div>
```

Comment:
Since both use a container as the outer element, it could look something like below so it doesn't need an extra div

```
<div className={style.container}>
  {perspective === "thirdPerson" ? (
    <div className={style.thirdPersonDetails}>
      ...
    </div>
  ) : [
    <div key="details" className={style.details}>
      ...
    </div>,
    <div key="meaning" className={style.meaning}>
      ...
    </div>
  ]}
</div>
```